src/ssb_project_cli/ssb_project/app.py

```diff
@@ -251,31 +286,38 @@ def create(
     if " " in project_name:
         raise ValueError("Spaces not allowed in projectname, use underscore?")
 
-    if not skip_github and not github_token:
+    if add_github and not github_token:
         raise ValueError("Needs GitHub token, please specify with --github-token")
 
     if not debug_without_create_repo:
-        if not skip_github and is_github_repo(github_token, project_name):
+        if add_github and is_github_repo(github_token, project_name):
             raise ValueError(f"The repo {project_name} already exists on GitHub.")
 
-    create_project_from_template(project_name, description)
+    if add_github and description == "":
+        description = request_project_description()
 
+    create_project_from_template(project_name, description)
     # Create empty folder on root
```

Comment:
I think these comments could be removed, they were just meant as TODOs during the Hack

Reply:
Removed all TODOs from hack